### DISCUSSION TITLE
Metabase ingestion support for owner metadata and usage type

### DISCUSSION THREAD

**Discussion by julien-alpaca** [open-metadata/OpenMetadata#16045] [General]
@ulixius9 Any chance we can have the owner field populated in the metadata ingestion of Metabase? I believe we could use the creator of the dashboard as the owner.
Also Is there any plan to add a ingestion of type usage for Metabase?

Thank you

**Comment by harshach** [comment]
@OnkarVO7 can you check this

**Reply by OnkarVO7** [reply]
I'll take a look

**Reply by pmbrull** [reply]
cc @OnkarVO7

**Reply by OnkarVO7** [reply]
Opened an issue here https://github.com/open-metadata/OpenMetadata/issues/16101
we'll pick it up for 1.5 release
